src/components/LabelValue.tsx

```diff
@@ -6,19 +6,14 @@ import { View, Text, StyleSheet } from "react-native";
 type Props = { label: string; value: string };
 export const LabelValueItem = ({ label, value }: Props) => {
   return (
-    <View style={styles.layout}>
-      <Text style={styles.label}>{label}</Text>
+    <View>
+      <Text style={labelStyle}>{label}</Text>
       <Text style={Typography.body1}>{value}</Text>
     </View>
   );
 };
 
-const styles = StyleSheet.create({
-  layout: {
-    marginBottom: 20,
-  },
-  label: {
-    color: ColorTheme.REDUCED,
-    ...Typography.body4,
-  },
-});
+const labelStyle = StyleSheet.create([
+  { color: ColorTheme.REDUCED },
+  Typography.body4,
+]);
```

Comment:
Minor nit: Other instances in the PR have single-style stylesheets wrapped in a top-level `styles` object. Ex: callsite would be `styles.label` instead of `labelStyle`.